.github/workflows/backend_unit_tests.yml

```diff
@@ -43,6 +43,8 @@ jobs:
         uses: actions/checkout@v4
       - name: Merge develop and set up dependencies
         uses: ./.github/actions/merge-develop-and-set-up-dependencies
+      - name: Install GraphicsMagick
+        run: sudo apt-get update && sudo apt-get install graphicsmagick
```

Comment:
won't this also need to be installed on developers' local machines when they run backend tests locally? This harkens back to my earlier comment https://github.com/oppia/oppia/pull/21694#discussion_r1989281478 which still needs to be addressed